### PR TITLE
feat: add a new plugin pub_med

### PR DESCRIPTION
**Pull request by LinkW77**
# Pull Request

## PR Description

Add a new plugin pub_med

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Performance enhancement
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other, please describe:

## Checklist

Before submitting this PR, please make sure:

- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) guidelines.
- [x] I have tested my changes locally to ensure they are effective.
- [x] I have updated the necessary documentation (if applicable).

### CODE REVIEW

File: plugin/bundles/pub_med/resources/i18n/en.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+bundle_name: "PubMed"
+bundle_description: "Search engine for biomedical literature."
+credential_pub_med_api_key_description: "API Key for authentication. Get it from https://account.ncbi.nlm.nih.gov/settings/"
+biomedical_search_input_query_name: "Query"
+biomedical_search_input_query_description: "The query to search for"
+biomedical_search_input_pub_date_name: "Published Date"
```
Comment (DynamesC) [inline plugin/bundles/pub_med/resources/i18n/en.yml:6]: rename it as pub_year

**Review by DynamesC — CHANGES_REQUESTED** [review]

**Review by DynamesC — APPROVED** [review]
LGTM

**Review by DynamesC — APPROVED** [review]
LGTM

**Review by taskingaijc — APPROVED** [review]
LGTM

**Pull request merged**